### ISSUE TITLE
Color codes do not work in Makefile

### ISSUE THREAD

**Issue by JakeRoggenbuck** [pavdmyt/yaspin#78]
In the Makefile, the color codes do not seem to work and do not output any color.

When running `make test`, the following is outputted.
```
\033[32;01m==> Cleaning bytecode ...\033[0m
\033[32;01m==> Linting code ...\033[0m
```

**Comment by pavdmyt** [comment]
Hi @JakeRoggenbuck 

It depends on terminal application you're using and it's settings. It was done for my convenience and absolutely not necessary. Works perfectly in iTerm2.

**Comment by pavdmyt** [comment]
Fixed by #81
